tests/cluster/schema_manager/test_schema_manager.py

```diff
@@ -5,14 +5,20 @@
 from atc.configurator import Configurator
 from atc.delta import DeltaHandle
 from atc.schema_manager import SchemaManager
-from tests.cluster.delta.SparkExecutor import SparkSqlExecutor
-from tests.cluster.schema_manager import extras
+
+from . import extras
+from .SparkExecutor import SparkSqlExecutor
 
 
 class TestSchemaManager(unittest.TestCase):
-    def test_get_sql_schema(self):
-        Configurator().add_resource_path(extras)
+    @classmethod
+    def setUpClass(cls) -> None:
+        c = Configurator()
+        c.clear_all_configurations()
```

Comment:
clearing the configurations is important if you start asserting stuff about all known keys.